### DISCUSSION TITLE
Can't submit issue at homebrew-core

### DISCUSSION THREAD

**Discussion by luckman212** [Homebrew/discussions#2505] [Everyday usage]
no matter what I do I am just getting an HTTP 400 error when trying to submit a bugreport at https://github.com/Homebrew/homebrew-core/issues/new/choose

![image](https://user-images.githubusercontent.com/1992842/142875082-592ff0f0-c255-4b99-889d-2fa5db2b44a3.png)

**Comment by carlocab** [comment]
Duplicate of #2465.

This isn't really something we can fix, since this seems to be happening on GitHub's end. I suggest filing a support ticket with GitHub.

**Reply by luckman212** [reply]
Okay thanks. Surprised that after 11 days GH hasn't addressed it. Will keep 👀 out. Here's a [link to the ticket](https://github.com/github/feedback/discussions/7788).

**Reply by luckman212** [reply]
I filed GH ticket #1409253

**Comment by luckman212** [comment]
Apparently this is fixable by https://github.com/Homebrew/discussions/discussions/2526#discussioncomment-1731168
